client/app/lib/components/extensions/conditions/conditions/SurveyCondition.tsx

```diff
@@ -67,7 +67,7 @@ const SurveyConditionForm = (
             {...field}
             disableClearable
             fullWidth
-            getOptionLabel={(id): string => surveys[id] ?? ''}
+            getOptionLabel={(id): string => `${surveys[id]} (${id})` ?? ''}
```

Comment:
I dont think we need to add the nullish coalescing operator here as `${surveys[id]} (${id})` will always return a value.